packages/webAuthn/background/webAuthn.ts

```diff
@@ -0,0 +1,226 @@
+import {
+  runtime,
+  tabs,
+  windows,
+  type Runtime,
+  type Tabs,
+  type Windows
+} from "webextension-polyfill"
+
+import json from "~packages/util/json"
+import { PortName } from "~packages/webAuthn/tabs/port"
+import {
+  isWebAuthnError,
+  type WebAuthnAuthentication,
+  type WebAuthnCreation,
+  type WebAuthnError,
+  type WebAuthnParams,
+  type WebAuthnRegistration,
+  type WebAuthnRequest
+} from "~packages/webAuthn/typing"
+
+export const webAuthnCreationAsyncWindow = async (
```

Comment:
nit: I could be better to name function as action prefixed with verb. For example `createWebAuthn...`

And I'm also pondering that does developer need to know this function actually creates a window? 🤔 We can discuss it later today